src/options/Options.vue

```diff
@@ -210,8 +210,8 @@ const faviconOptions: Array<{
   value: IconsTypeSettings;
 }> = [
   {
-    label: lang.favicon.global,
-    value: 'global',
+    label: lang.favicon.globe,
+    value: 'globe',
```

Comment:
todo: Changing the value will cause issues when loading a backup file

There are two options to prevent this issue :
- Keep the previous value
- Create a migration script that will be executed whenever a backup is loaded. The system is already in place, we just need a new entry in `src/logic/settings-migration.ts`.

Reply:
Nice, thanks for the tip! I'll try to address this as soon as I'm able :)

Reply:
@Elliot67 I've updated the PR. Not sure if I added the migration script correctly, please let me know if there's anything missing or incorrect!